lib/players/computer.rb

```diff
@@ -0,0 +1,122 @@
+require 'pry'
+
+module Players
```

Comment:
this class should have access to the game board and be able to get the win_combinations from that